src/python/views/demo_tigergraph_circle/__init__.py

```diff
@@ -0,0 +1,774 @@
+import asyncio
+import graphistry
+import os
+import pandas as pd
+import streamlit as st
+from components import GraphistrySt, URLParam
+from neptune_helper import gremlin_helper, df_helper
```

Comment:
the neptune + gremlin helpers seem to be copy-pasted -- we should remove for pedagogic purposes

Reply:
All packages not being used have now been removed